src/provider/view/setup.ts

```diff
@@ -0,0 +1,131 @@
+import {CHANNEL_NAME} from '../../client/models/config';
```

Comment:
Any reason this file is using two-space indenting?

Reply:
Linting rules/service-tools hasn't been setup yet.

Reply:
Oh sure, but still think it's worth getting stuff like this right. We're being generally consistent with indenting elsewhere even without linting